sim/scenarios/corrupt-rate/corrupt-rate-error-model.cc

```diff
@@ -58,6 +58,7 @@ bool CorruptRateErrorModel::DoCorrupt(Ptr<Packet> p) {
         payload[pos] = n;
         break;
     }
+    cout << "Corrupted packet (" << qp.GetUdpPayload().size() << " bytes) from " << qp.GetIpv4Header().GetSource() << endl;
```

Comment:
This is a duplicate log message. Can we merge this into the previous one?

Reply:
done.